optimizely/helpers/condition.py

```diff
@@ -19,6 +19,8 @@
 from . import validator
 from .enums import CommonAudienceEvaluationLogs as audience_logs
 
+class OptimizelyErrors():
+    AttributeFormatInvalid = "Provided attributes are in an invalid format."
```

Comment:
We normally add these constants to enums.py. Use an existing or add a new message in there. https://github.com/AmnaEjaz/python-sdk/blob/master/optimizely/helpers/enums.py#L96